Input/InputDecoratorIcon.js

```diff
@@ -58,7 +65,8 @@ const InputDecoratorIconBase = kind({
 	computed: {
 		className: ({position, styler}) => {
 			return styler.append('icon' + (position === 'before' ? 'Before' : 'After'));
-		}
+		},
+		size: ({size, skin}) => (size || skin === 'silicon' ? 'small' : 'large')
```

Comment:
@seunghoh shared the size default value of the Button yesterday.
Like the Button, I hope that Input also have the `large` size as a default value for `silicon` skin. In addition with the `small` size in `silicon` we could meet the GUI requirement.
Any questions, please let me know.
Sorry for the inconvenience.

Reply:
To get this right... I need to set the size of `input field` to `large` in `silicon` and keep the size of `input icon` to small. Do I get it right?

Reply:
Hello @adrian-cocoara-lgp 
I'm not sure I understand what you mean.

Could I explain my comment above?

> Like the Button, I hope that Input also have the large size as a default value for silicon skin. In addition with the small size in silicon we could meet the GUI requirement.

In Input\Input.js of the `develop` branch, the `size` prop could have `small` or `large` and there is no default value for the `size`. It's perfect. You don't have to fix anything in this PR.
```
		/**
		 * The size of the input field.
		 *
		 * @type {('small'|'large')}
		 * @default 'large'
		 * @public
		 */
		size: PropTypes.oneOf(['small', 'large']),
```

In Input\InputDecoratorIcon.js of the `develop` branch, the `size` prop could have `small` or `large` and the `size`'s default value is `large`. It's perfect for now. You don't have to fix anything in this PR, too.
```
		/**
		 * The size of the icon.
		 *
		 * @type {('small'|'large')}
		 * @default 'large'
		 * @public
		 */
		size: PropTypes.oneOf(['small', 'large'])
	},

	defaultProps: {
		size: 'large'
	},
```

For app developers using `silicon` skin, they should use the `size` prop with `small` to meet the GUI requirement.
```
<Input size="small" ... />
```
Then, the `icon` with 40px x 40px in the `input` should show based on the GUI. You might handle the size properly in this PR.